Multisig/UI/WalletConnect/Server/WalletConnectServerController.swift

```diff
@@ -76,6 +76,10 @@ class WalletConnectServerController: ServerDelegate {
         preconditionFailure("Subclass should override createSession method")
     }
 
+    func server(_ server: Server, didReceiveConnectionRequest requestId: RequestID, for session: Session) {
```

Comment:
let's move this into a new protocol so that we don't have to change existing api